docs/src/test/java/docs/home/persistence/CqrsIntegrationTest.java

```diff
@@ -71,7 +71,7 @@ public static void setup() throws Exception {
 
     application = new GuiceApplicationBuilder()
         .configure(config)
-        .bindings(bind(ServiceLocator.class).to(NoServiceLocator.class))
+        .bindings(bind(ServiceLocator.class).to(ConfigurationServiceLocator.class))
```

Comment:
Tests don't usually hit a `ServiceLocator` since they use injected settings to locate `Cassandra` and test code only runs a single service (so no need to locate other services).

I don't understand this change.